### DISCUSSION TITLE
Add the possibility to jump back in the code

### DISCUSSION THREAD

**Discussion by Cronos87** [godotengine/godot-proposals#7804] [Editor]
Adding the possibility to jump back in the code editor would be an excellent enhancement for people who like to code mainly using the keyboard.

This feature can be tested in VSCode by pressing the `CTRL + -` shortcut. Doing so, the editor will bring you back to the code you were editing before. The contrary is also true; pressing `CTRL + SHIFT + -` will jump forward in the code history.

Here is an article speaking about this: https://bobbyhadz.com/blog/vscode-go-to-last-edit-location#got-to-the-previous-or-next-location-in-vs-code

Keep in mind it is just a suggestion to enhance the coding experience for people like me who like to use only the keyboard 😁

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/4989.

**Comment by theraot** [comment]
For me (on WIndows) it works already with `ALT + LEFT_CURSOR_KEY` and `ALT + RIGHT_CURSOR_KEY` in Godot, VSCode, and most browsers.

I take you are on macOS given the shortcut you suggest and the linked article. I presume you can modify the actions in Godot to match.

---
Addendum:

![image](https://github.com/godotengine/godot-proposals/assets/1212509/da5e1eac-9cce-4c5e-bf6b-f211a6a3bfef)


There.

**Reply by Cronos87** [reply]
That's really cool! I didn't know about it! Thank you a lot for the help!
